### PR TITLE
[KIECLOUD-248] add ui link to csv

### PR DESCRIPTION
**Pull request by tchughesiv**
Dynamically detects its own CSV and updates the links section with the appropriate link/route to the console UI.
```
{"level":"info","ts":"2019-06-07T21:21:13.251242485Z","logger":"kieapp.controller","msg":"Updating kiecloud-operator.1.1.1 ClusterServiceVersion."}
``` 
Screen Shot 2019-06-07 at 4 21 44 PM

/assign @bmozaffa @sutaakar 

Signed-off-by: tchughesiv <tchughesiv@gmail.com>

### CODE REVIEW

**Comment by bmozaffa** [comment]
Looks good, let's just wait to hear on the wording / syntax, as well as which release to include this in.

**Comment by sutaakar** [comment]
@tchughesiv Would it be possible to add a test checking that link is correctly generated?

File: deploy/catalog_resources/redhat/1.1.1/businessautomation-operator.1.1.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -18,7 +18,10 @@ metadata:
     operator-businessautomation: "true"
 spec:
   displayName: Business Automation
-  description: Business Automation Operator for deployment and management of RHPAM/RHDM environments.
+  description: |
+    Business Automation Operator for deployment and management of RHPAM/RHDM environments.
+
+    **Look left for a link to the Installer UI.**
```
Comment (bmozaffa) [inline deploy/catalog_resources/redhat/1.1.1/businessautomation-operator.1.1.1.clusterserviceversion.yaml]: **To use the guided installer to provision an environment, open the Installer link, in the links section on the left side of this page**

**Comment by tchughesiv** [comment]
@sutaakar unit tests added by @bmozaffa

File: deploy/catalog_resources/community/1.1.0/kiecloud-operator.1.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -18,7 +18,10 @@ metadata:
     operator-kiecloud: "true"
 spec:
   displayName: Kie Cloud
-  description: Kie Cloud Operator for deployment and management of RHPAM/RHDM environments.
+  description: |
+    Kie Cloud Operator for deployment and management of RHPAM/RHDM environments.
+
+    **To use the guided installer to provision an environment, open the Installer link, in the links section on the left side of this page.**
```
Comment (mramendi) [inline deploy/catalog_resources/community/1.1.0/kiecloud-operator.1.1.0.clusterserviceversion.yaml]: Remove comma after "Installer link", please.

Also this text is used even before subscribing, when there is no installer link. I suggest adding: "After subscribing to the Operator, to use the..."

**Pull request merged**